client/main.go

```diff
@@ -99,7 +98,7 @@ func main() {
 	}
 	defer file.Close()
 
-	logger = log.New(file, "operations:", log.LstdFlags)
+	logger = log.New(file, "--- name: "+name+" >> ", log.LstdFlags)
```

Comment:
```suggestion
	logger = log.New(file, fmt.Sprintf("--- name: % >> ", name), log.LstdFlags)
```

I feel string formatting could be cleaned up like this.

Reply:
Resolved in https://github.com/burntcarrot/rowix/pull/12/commits/3ac86c20a341238c4e082e99c18da81a175ca79a.